telemetry/derive/tests/ui_fail/non_snake_case_name.rs

```diff
@@ -0,0 +1,16 @@
+use iroha_core::wsv::{World, WorldStateView};
+use iroha_telemetry_derive::metrics;
+
+#[metrics(+"test query", "another_test_query_without_timing")]
+fn execute(wsv: &WorldStateView<World>) -> Result<(), ()> {
+	// TODO: AFAIK prometheus is fine with spaces in names, but
+	// enforcing a consistent style might be a good idea long-term.
+
+	// Uncomment this, when the test can be un-ignored
+	
+	// Ok(())
+}
+
+fn main() {
+	
+}
```

Comment:
I am somewhat against template tests. Who will remember to look this up? Instead of removing it I would side with being more strict and disallowing spaces until there is a requirement to enable them